envoy/api/v2/cds.proto

```diff
@@ -356,6 +356,9 @@ message Cluster {
     // .. note::
     //   The specified percent will be truncated to the nearest 1%.
     core.Percent healthy_panic_threshold = 1;
+    //Configuration for ZoneAwareLoadBalancer
```

Comment:
Each item needs full documentation, with references back into the primary documentation where appropriate. (In this case potentially referring to the parallel runtime configuration). Please also discuss what the defaults are. Please also hide with "not-implemented-hide" (see other examples) unless the Envoy PR will be ready for review at the same time.

Reply:
Thanks for the review . Pushed not-implemented-hide now . If i am not wrong https://github.com/envoyproxy/data-plane-api/blob/master/docs/root/intro/arch_overview/load_balancing.rst#zone-aware-routing is the right place to add these config options right? 
Regarding defaults : They were already mentioned as 100% for zone_routing_enabled and 6 for min_cluster_size , would you like to comment on that here?
Thanks in advance!